tekton/pipeline_run.go

```diff
@@ -37,34 +36,6 @@ import (
 // PipelineType represents a PipelineRun type within AppStudio
 type PipelineType string
 
-const (
-	// appstudioLabelPrefix is the prefix of the application label
-	appstudioLabelPrefix = "appstudio.openshift.io"
-
-	// pipelinesLabelPrefix is the prefix of the pipelines label
-	pipelinesLabelPrefix = "pipelines.appstudio.openshift.io"
-
-	// releaseLabelPrefix is the prefix of the release labels
-	releaseLabelPrefix = "release.appstudio.openshift.io"
-
-	//PipelineTypeRelease is the type for PipelineRuns created to run a release Pipeline
-	PipelineTypeRelease = "release"
```

Comment:
David told me that we were only moving labels and annotations to metadata, and other constants, such as this one which describes a type, would remain where it is. Not sure if that changed after I talked to him, it wasn't conveyed to you, or including this was a mistake

Reply:
You are right. It was basically a full cut and paste. Did not pay attention that this one is not a label.